### PR TITLE
Story9

### PR DESCRIPTION
**Pull request by TobySTW**
s9t2 - added profile link HTML to bottom of applicant details
s9t3 - added working hyperlink to the profile link (takes to dynamic profile page)
s9t4 - added clipboard icon which copies the link to the clipboard + shows when it has been clicked

### CODE REVIEW

File: templates/partials/applicantInfoModal.phtml [hunk 1]
```diff
@@ -69,6 +69,11 @@
                         <p class="detail">Portfolio Url: <span id="portfolioUrl"></span></p>
                         <p class="detail">Plesk Hosting Url: <span id="pleskHostingUrl"></span></p>
                         <p class="detail">Github Education Link: <span id="githubEducationLink"></span></p>
+                        <p class="detail">Profile Link: <a id="userProfileLink" href="userProfileLink"></a> <button class="clipboard" onclick="copyToClipboard('#userProfileLink')"> <i class="glyphicon glyphicon-copy"></i>
```
Comment (ashleycoles) [inline templates/partials/applicantInfoModal.phtml:72]: Can you make the link open in a new tab please

Comment (ashleycoles) [reply templates/partials/applicantInfoModal.phtml:72]: Also inline event listeners are the devils work - please remove the onclick here and add an event listener in the JS instead

File: public/js/applicantModal.js [hunk 2]
```diff
@@ -40,11 +40,19 @@ function addCurrency(number) {
...
+    $temp.val($(element).text()).select();
+    document.execCommand("copy");
+    $temp.remove();
+    document.querySelector("button.clipboard").innerText = 'Copied';
+}
 
 $(document).ready(function(){
     $(".myBtn").click(function(){
         var url = './api/getApplicant/' + this.dataset.id
+        let studentUrl = 'http://localhost:8080/public/'+this.dataset.id
```
Comment (ashleycoles) [inline public/js/applicantModal.js:55]: Switch to a var for consistency sake - even though let is better

**Review by ashleycoles — CHANGES_REQUESTED** [review]

**Review by ashleycoles — DISMISSED** [review]

File: public/js/applicantModal.js (hunk 2 above)
Comment (mporam) [reply public/js/applicantModal.js:55]: disagree, swap them both to lets, better than they found it over consistency

**Review by mporam — APPROVED** [review]

**Pull request merged**